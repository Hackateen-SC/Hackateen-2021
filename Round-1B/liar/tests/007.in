1000
870626681 120909222 280505660 834217695 886747745 537307022 335804578 535807595 798436742 672821070 31930226 923551776 678117327 960638889 378512002 17932760 448676989 973552097 667902401 733625600 838752940 483863068 524780530 425963264 810759370 300753892 250434642 976202765 287918755 473452439 509695988 374262775 241062900 468955545 535607439 30601033 791435786 11330044 888745547 757829681 711358877 784841488 919521380 157479931 504107669 512538268 683883620 835913203 309583116 244825696 992788253 473257292 466902392 66615432 549783088 214694968 784032863 890791618 814347587 755957003 20767198 481046845 968942280 737014191 149248308 735881342 92203690 769930419 931898753 427558249 848128604 551951205 481883414 584069932 608718076 398934118 440727681 353832037 568340271 651554516 540773480 893576274 756307299 202862649 822460321 367720143 758777472 786047524 226892357 292233131 48524903 820896314 935714486 379634760 680571115 420097405 250491485 640814837 67203776 581430469 136578774 702488323 624230724 415225942 726073859 658755894 61741245 17978279 318870581 448376994 65727657 959224285 126159823 709485588 223496773 355488528 776684299 642953196 897208524 61676652 253757678 527113909 622354933 485067529 22750556 300199436 7783870 575972674 422929017 898595824 265022252 701314059 188349116 205097271 456876404 544587269 214391483 570837473 60865497 626762422 844136588 473000051 248509049 761070704 782177043 314018295 522567863 115385956 237604147 385795244 764550790 179806042 552723659 14916795 512243700 337179548 60567517 696789447 58329095 555810790 530374340 394894876 490911690 310842322 821767516 649809524 807195658 578404385 896409214 100753084 898988117 770112489 591513290 206757276 292091845 52463039 741189908 623245581 337107635 465604494 822336556 568737719 465092561 366501384 83164035 724602199 653752329 991706367 529632800 641460425 465529044 151887638 147845412 593789808 958175952 448873439 645722685 370938514 949237892 163608852 9608873 707509467 968369106 848591027 86213362 900684150 468765877 853061130 430708350 373666488 387592931 969016568 562455122 200613632 400284375 12532946 857513844 802146498 243932443 433095824 202207043 861580065 952975242 399915692 279158680 550294959 736470319 756435166 250395058 820817994 502138198 558983104 526118064 754702826 189859318 395067402 471595449 850746970 767689541 169308392 976469615 566696306 410590166 144905271 658520174 478035881 584320615 649098783 389248593 798932929 478647248 310035856 987782497 88009432 493292995 762430413 240117105 647922339 784799860 574859759 392521474 944836507 906698391 508274076 713858790 36597616 978541894 71081473 396576976 818749464 921612897 235550482 915081211 748083633 272712634 804605451 305188243 30284155 994840295 163732832 300645159 420092590 894043040 531676179 209218109 585326440 599137494 82112543 82738143 843688676 684132901 656407329 307060225 716779582 552015973 658141182 497898982 371938605 248291544 791843396 65780494 20412394 571335979 622220157 676161535 623148707 193835875 361225894 973086050 71398482 98461985 47871488 380741704 528495405 958637594 145808321 48339598 12148639 366461936 46166904 124992868 81299988 364024444 75201488 94235348 412470068 630593497 624743565 797669890 195891684 682603504 600368838 176608495 59921271 317052842 53556004 488317453 761838998 732105341 214398680 697231117 53521682 919024236 533574980 502057986 652029563 605030179 866539305 240499075 225907711 464431995 525966938 990765819 190061436 323935926 540677821 918254969 21281993 211199776 376156801 354615146 988617117 828593065 877597377 315453314 495628286 785129267 284880929 651510311 973224106 423337036 481151233 683091281 917660064 992172915 220570706 640356451 941851422 142790415 440341131 399331007 709825951 509475087 473655153 383310281 770654060 274176896 169844663 245936684 50770424 137205317 393762975 709157103 820855052 948261683 187236828 673706270 373228622 284744798 374574729 806552299 58818930 476334298 790853474 14270572 869209447 74107027 370683453 790191703 830301933 928789213 486738265 304780770 369619366 958443486 616981072 46540606 372271315 552246411 908528975 425641655 490679708 607763691 28953539 122786463 381020690 948783236 909748506 577939419 702760008 72081832 725134031 766243527 353273021 310862442 146590724 122480376 352278316 233626104 835463389 921375879 699526209 565240156 8486142 907309162 327109153 346555102 810033146 59917347 188257244 793275409 655816650 365004133 241729653 646893841 664443635 203707286 596255837 453268986 900005021 211485978 336751712 973365839 667557897 46568288 306520749 857252327 135787048 163997765 141991396 271061780 8077508 357862351 255465174 137392405 751859960 349346615 990276804 488331101 338478145 71349537 898717551 815611847 314025218 995104994 968583847 722465798 833366190 105178105 163831225 616134698 803389041 136050320 579862265 61698317 741566507 889142283 850364226 192120249 218438211 768377276 916019565 702911668 695458693 784724188 384401435 789957470 430396754 148325622 216578611 510109307 973978762 272125423 465481915 582975959 511869896 101415918 667466517 917735245 45000313 332297592 553324153 544937856 364354418 43351879 674479267 241635924 574915291 547049031 321971673 326258390 844940051 656060607 757043106 34920628 584387564 894178359 971340386 1204393 759808142 276542522 468367266 950023939 866518279 602545179 810496715 203137301 418995348 364930556 452547528 423038496 344445380 636983432 895968635 481435171 401856968 997465034 769966404 746983536 735364447 372102800 342363285 163044093 885768890 534716531 773018344 168350599 789727925 403158829 600740623 950100694 108137534 894706198 990730620 838054629 116885273 179461618 987202929 259948165 418617614 508132445 256812220 462934587 927581149 335443984 173549654 453282545 271294459 373980960 188651208 423819458 596831121 288864794 622331478 695028573 485623031 555728387 220691717 615986361 277687389 97395747 651586299 145372586 173480805 123921616 231318067 129889605 954911900 122846983 835634971 972009887 306572683 800651912 727177450 945216616 524811455 532046063 205200905 759591842 684465916 568807125 96665194 694486959 166552207 148824763 165689204 231523146 976602088 894689986 342155359 955826474 187692030 431168654 676559431 561725099 76393558 290604979 473896231 738880013 863468199 684541319 173352578 397190050 761283480 305146476 783558973 774463710 109840497 156880277 5855576 795133188 514788307 9905986 605293025 576601811 732403610 120117966 954376313 126030224 721160456 785025517 503983052 985432243 520889230 218831815 348765618 111390566 333896242 596494994 50252008 465722392 320271030 910921898 79588263 309220030 320099400 297097669 36193671 226548187 602542267 929306185 638847289 432460162 496785120 727557286 6384541 871239059 685830460 648994603 295930741 128873306 43108357 997322879 217672743 460142401 306598603 471824258 525472387 846098806 884479966 13913690 704800479 214908548 201799080 45620122 513266236 898711146 868370959 757439688 691417368 635125233 940912226 692920437 996655694 538042506 174127065 514266794 568876772 716945524 953654830 387938762 915733983 212578477 305040466 419513330 926948437 273564138 525725704 99809991 183231937 947911261 349002288 102175195 160247298 996024802 174506325 912037815 527732454 787454893 284750569 736798652 274527936 878486609 751300613 904216046 382224370 47638677 249600623 732897071 518956638 58466111 718597080 81852242 605041783 911958808 136241829 309877074 582994855 830873943 213130075 793807811 827589960 215181110 33886902 193974141 417239115 276755156 488829294 98991500 266327958 820246219 430663367 106626614 707753128 478450887 46299896 728585016 166118012 425130235 887254406 35258459 621722350 191292314 819355885 759009506 838128469 268434359 554357051 135350451 17177447 482151132 163238839 607320510 480510203 172807936 994438055 391161444 880013489 887406830 915718407 665435763 561704962 990839204 757424285 495046784 279711911 821826984 192358330 686752054 664305692 487014027 400498240 321187328 15623851 268021015 84581027 795672595 102687807 538851668 63020196 758745640 23587632 893417374 830183578 757078229 247413182 114669176 444254144 153821251 541017077 557473751 68188027 681093298 244438957 307131800 703582268 164501541 840204665 50873952 937827221 176028845 550083623 189702262 252470443 417921712 471979584 715665631 379439069 331724212 256604641 686304554 458415512 152118410 628111820 892401190 128751397 157205522 358921171 605112933 662625307 336067951 874735833 803687679 392032497 444580577 748114752 146993429 799892372 55773920 662977411 884999300 85099599 782151755 81256367 879963668 249677856 550996597 241493139 194169347 399105893 936368477 459982014 938905472 588465264 173179646 483276911 31183179 265597694 643476999 144272785 957278443 413411065 601105313 996950309 534006717 217414769 283569024 684054608 832469424 471709871 494732780 733440631 19446519 720237739 354554732 857644330 723244815 400515853 903223144 338843460 438094655 921156260 937241359 802078455 102283032 193683934 518473946 70085179 888169279 656558605 439251596 111499949 256826469 764665499 675025030 591044218 957147287 140162845 173631166 35583969 80760853 406171048 377737576 997188502 266545113 405665410 577420700 468333563 858735947 625984682 89174054 351756131 742043442 601340681 135589375 122120945 659467711 756295423 182816311 978776668 98741837 218445027 303992647 251632942 422180269 550288925 996013345 215405081 192427064 311008539 825027359 423361070 572102080 405868148 959126978 148799921 186567824 790531938 223082479 740295668 389054439 1620698 126118687 427147254 860546316 898533255 748417737 374657383 748491983 529560938 515062253 713695941 810267869 873178149 390962357 559548150 279046602 107596264 361755472 209484373 676478569 109975724 253007963 740499039 535628572 874653297 51524986 473749757 603552607 378493792
GGGGGGLLLLLLLLGGGGLLGLGGLLLGLLLLGLGLGGLLLGGLLLGGGLGLGLLGGLLGGLGGLGGLGGLLGLGLLLLLGLLLGLLLLGGGLGLLLLGGLGGLLGLLGLGGGGLLLLGLLGGLGLGGGGGGGGGLLLLGLLLGGGGGLLLLLGLGLLGLGLGLLLGGLLLGLLLLLLLLGLLGGGLGGLGLLLGGLLGLLLGLLLGLGLLGGGGGGLLGGLGLGLGLLLGGGLLGLGLLLLLGGLLGGGLGGLGGGLLLGGGLLGLGGGLGGLGLLLLGGGGLGLLGLLLGGLGGGGLLLGGGGLGGGLLLGGGLLLLLLLGGLLGLLGLLGLLGLGGLLGGGGLLLGGLGGLLLLLGLLGLGGGLLGGLGLLGLLGGGLLGGLLGGLLLGGGGGGLGLGGLGGGGGLLLLLLGLGGGLGGLLLLLLGLGGLLGLLLLLGLGLLLLGLLLGGLGLLLLGLLGGGGGLGLLLLLGLLGGLLGLGGGGGGGGLLLLGLGLLLLLLLLGGGGGGGGLGLGGGGLGLGGLGGGLLLLGGGGGGGLLGLGGGLLGLGLGLLGLGGGLGGGLGLGLGGLLLGLLGGLLLGGLLLLLGLLGGLGGGGLLGLLGGLGGGLGGLGLGGLLGGLLLGLLGLLGLLLLGLGLGGGGLLGGGLGLGGLGLLGLLGGGLLGGGGGGGLGLGGLGGGGLGGGLGGLLGGLGLLGGLGLGGLLLGGLLGGGGGLGGGGGGGLGLLGGGLLGLLGGGGGGGGGGGLLGGGLLLGLLLGLLGLLLLGGLGGGGLGGGGLLLLLGLLGLGLLLGGLLLGGLLGGGGLGGLGGLGLLLLLGLGLGLGGGLLGLGLLLGLGGGLGLGLGGLGGGGGLLLGLLLLLLGGLLGLGLGGLLGLGGLGLLGLLGLGLLLLLLLLLGGLGGGGGGLGGLGLGGLGLGLLGLLGGGGGGGGLGGGLGLLGLGLGLGLGLGLLLLGLGGGLLLLGGLLLLLLLGLGLGGLGGLLLGLGGGGGGLLL
