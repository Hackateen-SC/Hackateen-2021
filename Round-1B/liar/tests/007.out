58329096 491
